pytorch/pytorch-engine/src/main/java/ai/djl/pytorch/engine/PtNDArray.java

```diff
@@ -265,6 +265,12 @@ public void set(Buffer data) {
         JniUtils.set(this, buf);
     }
 
+    /** {@inheritDoc} */
```

Comment:
If we put it here, it may not activate for some calls to get that use the NDIndex directly. The more precise place would be to integrate it into the NDArrayIndexer. Maybe add a new NDIndexFull type to handle this?

Reply:
I checked it but can't come up with a good way to avoid NDIndexFullPick.fromIndex and NDIndexFullSlice.fromIndex. Maybe we can merge it for now and later optimize it?

Reply:
It should avoid fullpick on its own because pick requires a pick NDIndexElement, while this is a fixed. Avoiding the slice is difficult, but there is an order of trying the NDIndexFull* inside the `NDArrayIndexer`. Just make your new one before `NDIndexFullSlice` in the order. In fact, maybe just make your new one first in the order. You may also need to throw an exception in the FullFixed get and catch it in the NDArrayIndexer get to make the FullFixed handling optional.

If you prefer, go ahead and merge it and then I can take a look at it later.

Reply:
ok Thank, I will go ahead to merge it and you can come back to this when you have time